app/src/components/header/index.tsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
-import { Box } from 'lib-react-components';
+import { Box, Typography } from 'lib-react-components';
 import Link from 'next/link';
 import PropTypes from 'prop-types';
 import clx from 'classnames';
+import { Sidebar } from '../../pages/docs/sidebar';
+import config from '../../../sources/docs/config.json'
 import s from './index.module.sass';
 
 interface IHeaderProps {
   isMain?: boolean;
 }
 
-export const Header: React.FC<IHeaderProps> = (props: IHeaderProps, context) => {
+export const Header = (props: IHeaderProps, context) => {
+  const [showSidebar, setShowSidebar] = React.useState(false);
```

Comment:
Don't need to add this solution to PR with desktop styles. Move it to another PR.